python-frontend/src/main/java/org/sonar/python/types/TypeShed.java

```diff
@@ -477,12 +477,6 @@ public static Symbol symbolWithFQN(String fullyQualifiedName) {
     if (builtinSymbol != null) {
       return builtinSymbol;
     }
-    for (Map<String, Symbol> symbolsByFqn : typeShedSymbols.values()) {
-      Symbol symbol = symbolsByFqn.get(fullyQualifiedName);
-      if (symbol != null) {
-        return symbol;
-      }
-    }
     String[] fqnSplittedByDot = fullyQualifiedName.split("\\.");
     String moduleName = Arrays.stream(fqnSplittedByDot, 0, fqnSplittedByDot.length - 1).collect(Collectors.joining("."));
     Set<Symbol> symbols = symbolsForModule(moduleName);
```

Comment:
On line 484 (which I can't highlight as it wasn't changed), shouldn't we try to find a symbol with a given `name` instead of `fullyQualifiedName`?
So that we can retrieve symbols from "façade" modules that export imported symbols whose actual FQN might be different?
Something like that is done on line 190-193 on the other `symbolWithFQN` method.

Reply:
I think it makes sense indeed. The only concern might be on performance, as we'd have to iterate on all the symbols of a module. We should probably optimize map lookup by name instead of having fqn as key

Reply:
Indeed. I guess it can also be done later and just have some TODO + referencing an appropriate ticket for that, given this PR is really about fixing a inconsistency bug.